### PR TITLE
Improves supermatter logging

### PR DESCRIPTION
**Pull request by Tacolizard**
Added logs (with player names) when the SM is attacked in various ways. This will help investigating how exactly the SM was damaged.

I don't know how exactly to do this and I don't feel like reading the blue book or going on the wiki so I'm just using process of elimination until the errors die.

### CODE REVIEW

**Comment by Tacolizard** [comment]
whatevs

**Comment by PKPenguin321** [comment]
? Please fix this

**Comment by Tacolizard** [comment]
i will when i feel like it

File: code/modules/power/supermatter/supermatter.dm [hunk 1]
```diff
@@ -486,6 +494,7 @@
 		return
 
 	playsound(get_turf(src), 'sound/effects/supermatter.ogg', 50, 1)
+	investigate_log("has been bumped by [user].", "supermatter")
```
Comment (swindly) [inline code/modules/power/supermatter/supermatter.dm]: user is not defined.

Comment (Cyberboss) [inline code/modules/power/supermatter/supermatter.dm]: use [AM]

**Pull request closed without merging**